zeppelin-plugins/launcher/yarn/src/main/java/org/apache/zeppelin/interpreter/launcher/YarnRemoteInterpreterProcess.java

```diff
@@ -178,6 +178,10 @@ public void start(String userName) throws IOException {
       this.appId = appResponse.getApplicationId();
       ApplicationSubmissionContext appContext = yarnApplication.getApplicationSubmissionContext();
       appContext = createApplicationSubmissionContext(appContext);
+      String YARN_NODE_LABEL_EXPRESSION = "zeppelin.interpreter.yarn.node.label.expression";
+      if(properties.containsKey(YARN_NODE_LABEL_EXPRESSION) && StringUtils.isNotEmpty(properties.getProperty(YARN_NODE_LABEL_EXPRESSION))) {
```

Comment:
You can use `properties.getProperty(key, defaultValue)` to simplify it

Reply:
> You can use `properties.getProperty(key, defaultValue)` to simplify it
Yes, it's a good idea

Reply:
Done